static/flow/views/program-editor-panel.js

```diff
@@ -1361,23 +1368,6 @@ var ProgramEditorPanel = function(options) {
     //
     this.receivedSensorData = {};
 
-    //
```

Comment:
This function appears to be totally obsolete.  It's from before my time so I'm not entirely sure what it was for....